enclave-manager/web/src/components/AppLayout.tsx

```diff
@@ -0,0 +1,19 @@
+import { Flex } from "@chakra-ui/react";
+import React, { PropsWithChildren } from "react";
+
+type AppLayoutProps = PropsWithChildren<{
+  Nav: React.ReactElement;
+}>;
+
+export const AppLayout = ({ Nav, children }: AppLayoutProps) => {
+  return (
+    <Flex flexDirection={"row"}>
+      {Nav}
+      <Flex as="main" w={"100%"} justifyContent={"flex-start"} p={"20px 40px"} className={"app-container"}>
+        <Flex maxWidth={"1320px"} w={"100%"}>
```

Comment:
I think we are not making this the maxWidth right? 
But if there has to be a maxWidth can we pull it out into `constants.ts`?

Reply:
just checked on slack what we are doing here. I agree making this a constant seems reasonable.